src/main/java/AddListCommand.java

```diff
@@ -0,0 +1,18 @@
+class AddListCommand extends UserCommand {
+    private String addCommand;
+
+    AddListCommand (String addCommand, TaskList tasks) {
+        super(tasks);
+        this.addCommand = addCommand;
+    }
+
+    @Override
+    public void execute () {
+        System.out.println("     Got it. I've added this task: ");
+        int totalTasksNum = tasks.addList(this.addCommand);
+
+        System.out.println("     Now you have " + totalTasksNum +
+                ((totalTasksNum > 1) ? " tasks" : " task") + " in the list.");
```

Comment:
Good job on splitting the long lines into 2 individual lines but i think the indentation of the second line is not accurate